root/static/scripts/edit/components/HelpIcon.js

```diff
@@ -15,32 +15,24 @@ type PropsT = {
   +content: React.Node,
 };
 
+const ICON_STYLE = {
+  display: 'inline-block',
+  marginLeft: '10px',
+  verticalAlign: 'text-top',
+};
+
 const HelpIcon = ({
   content,
-}: PropsT): React.Element<'div'> => {
-  const [hover, setHover] = React.useState(false);
-  return (
-    <div
-      style={{
-        position: 'relative',
-        display: 'inline-block',
-        marginLeft: '10px',
-      }}
-    >
+}: PropsT): React.MixedElement => (
```

Comment:
This is just `Tooltip`? Same for at least `BatchCreateWorksButtonPopover` which I cannot leave a comment on for reasons.